src/main/java/bio/terra/cloudevents/CloudStorageEventType.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.cloudevents;
+
+/**
+ * For a enumeration of all Cloud Event types, please refer to
+ * https://cloud.google.com/functions/docs/calling/storage
+ */
+public enum CloudStorageEventType {
+  GOOGLE_STORAGE_OBJECT_FINALIZE(
+      "google.storage.object.finalize", "Google Storage Object Finalize"),
+  GOOGLE_STORAGE_OBJECT_ARCHIVE("google.storage.object.archive", "Google Storage Object Archive"),
+  GOOGLE_STORAGE_OBJECT_DELETE("google.storage.object.delete", "Google Storage Object Delete"),
+  GOOGLE_STORAGE_OBJECT_V1_METADATAUPDATED(
```

Comment:
Currently we only use `GOOGLE_STORAGE_OBJECT_FINALIZE`.  My preference would be to leave these other event types out until we need them. https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Reply:
Unused event types have been removed.